lib/hammer_cli_katello/repository.rb

```diff
@@ -255,6 +262,27 @@ def import_upload_ids(ids)
         }
         resource.call(:import_uploads, params)
       end
+
+      def progress_message(filename)
+        _('Uploading %{filename}') % {:filename => filename}
+      end
+
+      def render_result
+        print_message _('finished')
```

Comment:
Looks like the trend is capitalized words and what is the effect if errors occur during the upload process? Since the progress bar method is doing an 'ensure' this finished would get printed everytime, but will it be obvious there were errors and what they are?